docs/usage/configuration-options.md

```diff
@@ -1190,6 +1190,23 @@ A preset alternative to the above is:
 }
 ```
 
+To match specific ports you have to add a protocol to `matchHost`:
+
+```json
+{
+  "hostRules": [
+    {
+      "matchHost": "https://domain.com:9118",
+      "enabled": false
```

Comment:
It's not immediately clear why enabled=false in the config example.

I would expect to see a allowlist approach instead of a limited block list approach?

Reply:
This is a follow up of the example above. 
The setting could be anything what we support in hostRules.